app/views/reviews/edit.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= label_tag :title %>
   <%= text_field_tag :title, @review.title %>
   <%= label_tag :rating %>
-  <%= text_field_tag :rating, @review.rating %>
+  <%= select_tag :rating, options_for_select((1..5).to_a) %>
```

Comment:
Adds a clean look to the drop down for selecting a number 1-5 for review ratings